about.html

```diff
@@ -0,0 +1,120 @@
+                            <span class="bar"></span>
+                        </div>
+                    </div>
+                </div>
+                <div class="desk-navbar fixed-top">
+                <div class="black_nav_desktop">
+                    <div class="nav_logos_desktop">
+                      <a href="www.facebook.com"><img class="fb" src="images/facebook-new.png" alt="facebook logo"></a>
+                      <a href="www.twitter.com"><img class="twitter" src="images/twitter-new.png" alt="twitter logo"></a>
+                    </div>
+                    <a href="#">English</a>
+                    <a href="#">My Page</a>
+                    <a href="#">Korean</a>
+                  </div>
+                  <div class="desk-menu">
+                    <ul class="menu">
+                      <a href="index.html"><img class="logo-desk" src="images/lion-logo.png" alt="logo"></a>
+                      <li><a id="about-desk" href="index.html">Home</a></li>
+                      <li><a href="#welcome">Program</a></li>
+                      <li><a href="#welcome">Join</a></li>
+                      <li><a href="#welcome">Sponsor</a></li>
+                      <li><a href="#welcome">News</a></li>
+                      <li><a id="cc" href="#welcome">CC Campaign</a></li>
+                    </ul>
+                  </div>
+                  </div>
+            </header>
+            <div class="welcome-container-about d-flex flex-sm-column flex-column">
+            <h2 class="welcome-top top" id="welcome-about-page">Welcome all!</h2>
+            <h1 class="welcome fw-bold">TO THE GREATEST CONCERT WEEK</br>
+                <div class="break"> IN THE 254🔥</div>
+            </h1>
+            <p id="desc-about" class="desc-about border border-3 bg-white">
+                Attention music lovers!
+                We are thrilled to announce that our much-anticipated concert week is finally here! It's time to gather your friends, family, and loved ones for a week of unforgettable musical performances.<br />
+                From classical to contemporary, our lineup of talented musicians is set to entertain you with their incredible skills and passion for music. Join us as we celebrate the power of music and its ability to bring people together. <br />
+                This year's concert week promises to be a memorable one, with a range of musical genres to suit everyone's taste. Whether you prefer the soothing sounds of jazz or the upbeat rhythms of pop, we've got you covered. <br />
+                So, what are you waiting for? <br />Book your tickets now and get ready to be blown away by our amazing performers. It's a week you won't want to miss!
+                Don't forget to spread the word and invite your friends and family to join us for this incredible musical experience. <br />Let's make this concert week one to remember!
+                We can't wait to see you there!
+            </p>
```

Comment:
- [ ] As mentioned by the previous reviewer https://github.com/mnaibei/capstone_1/pull/1#discussion_r1133980512 please fix your description, currently it is taking too much space try reducing the font-size to 14px and giving a padding 0f 15 or 16px, I would also suggest to remove the text-align center property.

Reply:
done